From 2b1532d95b85f63c1e7e370660948604b7d5f576 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Tue, 10 Sep 2019 20:49:44 +0300
Subject: [PATCH 33/33] Enable usb ethernet dongle asix-88179

Enable the dongle. In order to use it issue:
usb start
setenv ethact ax88179_eth

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 configs/ucm-imx8m-mini_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/ucm-imx8m-mini_defconfig b/configs/ucm-imx8m-mini_defconfig
index 8dbf544e86..3d750f7fe1 100644
--- a/configs/ucm-imx8m-mini_defconfig
+++ b/configs/ucm-imx8m-mini_defconfig
@@ -52,6 +52,8 @@ CONFIG_DM_SPI_FLASH=y
 CONFIG_SPI_FLASH=y
 CONFIG_SPI_FLASH_STMICRO=y
 CONFIG_DM_ETH=y
+CONFIG_USB_HOST_ETHER=y
+CONFIG_USB_ETHER_ASIX88179=y
 CONFIG_PINCTRL=y
 CONFIG_PINCTRL_IMX8M=y
 CONFIG_DM_REGULATOR=y
-- 
2.11.0

